lib/models/repository-states/present.js

```diff
@@ -258,8 +257,8 @@ export default class Present extends State {
   // Remote interactions
 
   @invalidate(branchName => [
-    Keys.aheadCount.oneWith(branchName),
-    Keys.behindCount.oneWith(branchName),
+    Keys.statusBundle,
+    Keys.headDescription,
```

Comment:
When will fetch or pull invalidate the `describe` output? Does that happen if you've checked out a remote tracking branch directly... ?